### PR TITLE
fastify

### PR DESCRIPTION
**Pull request by marie-fourier**
- fastify
- used JsonRPCSigner in testingMode
- benchmark mode disables logging
- optimized docker image size

### CODE REVIEW

File: packages/api/package.json [hunk 1]
```diff
@@ -31,23 +31,20 @@
...
-    "body-parser": "1.20.1",
     "class-transformer": "0.5.1",
     "class-validator": "0.14.0",
-    "compression": "1.7.4",
+    "db": "*",
     "ethers": "5.7.2",
-    "express": "4.18.2",
-    "joi": "17.7.0",
+    "executor": "*",
+    "fastify": "^4.14.1",
```
Comment (ch4r10t33r) [inline packages/api/package.json:39]: Pls use a specific version of the package.

File: packages/api/src/server.ts [hunk 2]
```diff
@@ -1,104 +1,93 @@
...
-
 export class Server {
-  private app: Application;
+  private app: FastifyInstance;
 
   constructor() {
-    this.app = express();
+    this.app = fastify({
+      logger,
+      disableRequestLogging: true,
```
Comment (ch4r10t33r) [inline packages/api/src/server.ts:15]: Can this be configurable? Will be useful while testing/monitoring hosted instances, no?

**Review by ch4r10t33r — CHANGES_REQUESTED** [review]

File: packages/api/package.json (hunk 1 above)
Comment (marie-fourier) [reply packages/api/package.json]: done

**Review by ch4r10t33r — APPROVED** [review]
LGTM

**Pull request merged**